src/Modules/CalcDefence.lua

```diff
@@ -1397,28 +1401,37 @@ function calcs.defence(env, actor)
 		end
 	end
 	
-	-- Prevented life loss (Petrified Blood)
+	-- Prevented life loss taken over 4 seconds (and Petrified Blood)
 	do
-		output["preventedLifeLoss"] = modDB:Sum("BASE", nil, "LifeLossBelowHalfPrevented")
+		output["preventedLifeLoss"] = modDB:Sum("BASE", nil, "LifeLossPrevented")
+		output["preventedLifeLossBelowHalf"] = (1 - output["preventedLifeLoss"] / 100) * modDB:Sum("BASE", nil, "LifeLossBelowHalfPrevented")
 		local portionLife = 1
 		if not env.configInput["conditionLowLife"] then
 			--portion of life that is lowlife
 			portionLife = m_min(output.Life * data.misc.LowPoolThreshold / output.LifeRecoverable, 1)
-			output["preventedLifeLoss"] = output["preventedLifeLoss"] * portionLife
+			output["preventedLifeLossTotal"] = output["preventedLifeLoss"] + output["preventedLifeLossBelowHalf"] * portionLife
+		else
+			output["preventedLifeLossTotal"] = output["preventedLifeLoss"] + output["preventedLifeLossBelowHalf"]
 		end
 		if breakdown then
-			breakdown["preventedLifeLoss"] = {
+			breakdown["preventedLifeLossTotal"] = {
 				s_format("Total life protected:"),
 			}
+			if output["preventedLifeLoss"] ~= 0 then
+				t_insert(breakdown["preventedLifeLossTotal"], s_format("%.2f ^8(portion taken over 4 seconds instead)", output["preventedLifeLoss"] / 100))
+			end
 			if portionLife ~= 1 then
-				t_insert(breakdown["preventedLifeLoss"], s_format("%.2f ^8(initial portion taken from petrified blood)", output["preventedLifeLoss"] / portionLife / 100))
-				t_insert(breakdown["preventedLifeLoss"], s_format("* %.2f ^8(portion of life on low life)", portionLife))
-				t_insert(breakdown["preventedLifeLoss"], s_format("= %.2f ^8(final portion taken from petrified blood)", output["preventedLifeLoss"] / 100))
-				t_insert(breakdown["preventedLifeLoss"], s_format(""))
```

Comment:
I feel communication of where the 0.16 number comes from would be useful to display in the breakdown also show these two parts being added would also be nice along side how you subtract from one for the end number.
Your calculations all look correct to me though.
![image](https://user-images.githubusercontent.com/31533893/210067433-c256df2a-554e-435f-b6c2-dda18f3ccdb8.png)